editor/editor_themes.cpp

```diff
@@ -1215,7 +1215,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 
 	// Panel
 	theme->set_stylebox("panel", "Panel", make_flat_stylebox(dark_color_1, 6, 4, 6, 4, corner_width));
-	theme->set_stylebox("panel_fg", "Panel", style_default);
+	theme->set_stylebox("PanelForeground", "EditorStyles", style_default);
```

Comment:
Should be changed to use `SNAME`.

Reply:
Done. Didn't use it before because nothing else here did when making those changes.